kayak_core/src/color.rs

```diff
@@ -19,30 +19,27 @@ impl Default for Color {
 
 impl Color {
```

Comment:
We could, and that's what @IceSentry suggested as well. My hesitance with it was that we have an opportunity to do things in ways bevy doesn't. Their code is likely good enough, but maybe there'a a better library or method we could use that handles more or handles things better.

I don't know, that was just my reasoning at least 😅

Reply:
I'm just not sure how it works from a license point of view. If I just copy it all, should I add a comment pointing to the bevy Color impl?

Reply:
I'm not sure either to be honest. We've done it with other things (`KeyCode`, for example), but not something that contains a bunch of unique logic. I think it could fall under "re-distribution", meaning we might need to add a license? (For MIT at least)

Reply:
> I'm not sure either to be honest. We've done it with other things (`KeyCode`, for example), but not something that contains a bunch of unique logic. I think it could fall under "re-distribution", meaning we might need to add a license? (For MIT at least)

I think a lot of the keycode stuff was copied from winit as well, but it would probably be a good idea to include the bevy license somewhere. Perhaps as a link at the top of the file?